x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -37,6 +37,10 @@ export const entityColumnIdsRt = t.union([
 
 export type EntityColumnIds = t.TypeOf<typeof entityColumnIdsRt>;
 
+export const entityViewRt = t.union([t.literal('unified'), t.literal(ENTITY_TYPE)]);
```

Comment:
IMO you can choose from two options:

1. Use the group by to define which page you need to render:
```suggestion
export const entityGroupByRt = t.union([t.literal('none'), t.literal(ENTITY_TYPE)]);
```

`none` => unified view
`ENTITY_TYPE` => grouped view

2. Having a dedicated param to define the page:
```suggestion
export const entityViewRt = t.union([t.literal('unified'), t.literal('grouped')]);
```
If you go for this option you'll need another type to hold the group by field.

`unified` => unified view
`grouped` => grouped view

I would choose option [2], but it's up to you.